test/kafka.yml

```diff
@@ -0,0 +1,30 @@
+apiVersion: kafka.strimzi.io/v1alpha1
+kind: Kafka
+metadata:
+  name: my-cluster
+spec:
+  kafka:
+    version: 2.1.0
+    replicas: 1
+    listeners:
+      plain: {}
+      tls: {}
+    config:
+      offsets.topic.replication.factor: 1
+      transaction.state.log.replication.factor: 1
+      transaction.state.log.min.isr: 1
+      log.message.format.version: "2.1"
+    storage:
+      type: persistent-claim
+      size: 100Gi
```

Comment:
O_O
I hope it won't create a 100Gi volume?

Reply:
See comment above.

Reply:
OK, I was just afraid it would create a real 100Gi volume :-) If it doesn't, it's all good then